src-docs/spec/sendUpdate.md

```diff
@@ -35,6 +15,52 @@ In these cases, you can still call `sendUpdate()`,
 however, the update won't be sent to other peers
 and you won't get the update by [`setUpdateListener()`](./setUpdateListener.html).
 
+The `update` object has the following properties:  
+
+- `update.payload`: string, number, boolean, array, object or `null`.
+   MUST NOT be `undefined`.
+   Everything that is not JSON serializable will be skipped,
+   this especially affects Binary data buffers as used in `File`, `Blob`, `Int*Array` etc.;
+   if needed, use eg. base64.
+
+- `update.info`: optional, short, informational message that will be added to the chat,
+   e.g. "Alice voted" or "Bob scored 123 in MyGame".
+   Do not add linebreaks; implementations will truncate the text at about 50 characters or less.
+   If there are series of info messages, older ones may be dropped.
+   use this option sparingly to not spam the chat.
+
+- `update.notify`: optional dictionary mapping an [address](./selfAddr_and_selfName.md)
+  to a text that should be shown as a user-visible notification to the addressed user. 
+  The optional special key "\*" serves as a catch-all address 
+  whose text shall be notified if `selfAddr` is not contained in the dictionary. 
+
+- `update.href`: optional string that can be passed as the navigation
+  location to the webxdc app. 
+  If specified the string must start with `#` and can not contain any
```

Comment:
this restricts it too much and potentially make it hard for single-page apps with hash routers that want to use query part for a dialog while the hash is used for pages

Reply:
I think with some examples it is good enough to avoid confusion or force to always declare the path: index.html#fragment instead of allowing sending #fragment only